src/HW3/balance.c

```diff
@@ -0,0 +1,27 @@
+#include <stdio.h>
+#include <string.h>
+
+int main()
+{
+    int k = 0;
+    char strq[1000];
+    scanf("%s", &strq);
+    for(int i = 0; i < strlen(strq); i++){
```

Comment:
```suggestion
    for(int i = 0; i < strlen(strq); i++) {
```

Самостоятельно работать форматером --- плохая идея.